src/components/AvatarGroup/AvatarGroupCounterTooltipContainer.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useMemo } from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Dialog from "../Dialog/Dialog";
+import AvatarGroupCounterTooltipContent from "./AvatarGroupCounterTooltipContent";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+
+const AvatarGroupCounterTooltipContainer = ({
+  children,
+  avatars,
+  max,
+  type,
+  className,
+  counterTooltipCustomProps,
+  counterTooltipIsVirtualizedList,
+  counterTooltipTheme
+}) => {
+  const counterTooltipContentComponent = useMemo(() => {
+    return AvatarGroupCounterTooltipContent({
```

Comment:
that's not really standard way to write a component in jsx files. you can write the component with regular `<Component/>` syntax and no need to use useMemo here

Reply:
I'm using such syntax in order to obtain `null` from the `AvatarGroupCounterTooltipContent` (u can't get null from `<Component/>`), specifically in case when there is no avatars, so tooltip content couldn't be rendered. The other solution would be to just check avatars length at the `AvatarGroupCounterTooltipContainer, but I'm not sure that it'll be better there - what do you think?